v1/src/test/java/com/google/cloud/teleport/spanner/ImportFromAvroTest.java

```diff
@@ -1298,6 +1298,91 @@ public void models() throws Exception {
                 + " REMOTE OPTIONS (endpoint=\"//aiplatform.googleapis.com/projects/span-cloud-testing/locations/us-central1/publishers/google/models/textembedding-gecko\")"));
   }
 
+  // @Test
+  // public void propertyGraphs() throws Exception {
```

Comment:
Thanks for helping debugging this test. I've made the changes and enabled this test. It passes now.